src/Keboola/StorageApi/Metadata.php

```diff
@@ -112,6 +114,29 @@ public function postTableMetadata($tableId, $provider, $metadata)
         ));
     }
 
+    /**
+     * @param string $tableId
+     * @param string $provider
+     * @param array $metadata [['key' => 'some-key', 'value' => 'Some value'], ...]
+     * @param array $columnsMetadata ['columnName1' => [['key' => 'some-key', 'value' => 'Some value'], ...], ...]
```

Comment:
ten formát ide napsat tak aby mu rozumněl stan a pořádně ti napráskal když to poděláš. 
https://phpstan.org/writing-php-code/phpdoc-types#array-shapes

Reply:
jo, znal jsem jen ty `array<>`, tohle ne. Tak jsem to pridal.

Reply:
tady jsou pekny priklady: https://dev.to/suckup_de/modern-phpdoc-annotations-11d4#:~:text=1.2%20New%20(and%20not%20that%20new)%20Array%20Syntax